src/it/verify-it/verify.groovy

```diff
@@ -31,6 +35,34 @@ assert new File(basedir, 'target/generated-sources/localizer/org/jenkinsci/tools
 content = new File(basedir, 'target/generated-sources/localizer/org/jenkinsci/tools/hpi/its/Messages.java').text;
 assert content.contains(" holder.format(\"it.msg\");");
 
+assert new File(basedir, 'target/verify-it/META-INF/MANIFEST.MF').exists()
+
+InputStream is = Files.newInputStream(new File(basedir, 'target/verify-it/META-INF/MANIFEST.MF').toPath())
+try {
+  Manifest manifest = new Manifest(is)
+  assert !manifest.getMainAttributes().getValue('Build-Jdk-Spec').isEmpty()
+  assert manifest.getMainAttributes().getValue('Created-By').startsWith('Maven Archiver')
+  assert manifest.getMainAttributes().getValue('Extension-Name') == null // was provided by Maven 2, but core prefers Short-Name
+  assert manifest.getMainAttributes().getValue('Group-Id').equals('org.jenkins-ci.tools.hpi.its')
+  assert manifest.getMainAttributes().getValue('Hudson-Version').equals('2.249.1')
+  assert manifest.getMainAttributes().getValue('Implementation-Title').equals('MyNewPlugin') // was project.artifactId in previous versions, now project.name
+  assert manifest.getMainAttributes().getValue('Implementation-Version').equals('1.0-SNAPSHOT')
+  assert manifest.getMainAttributes().getValue('Jenkins-Version').equals('2.249.1')
+  assert manifest.getMainAttributes().getValue('Long-Name').equals('MyNewPlugin')
+  assert manifest.getMainAttributes().getValue('Manifest-Version').equals('1.0')
+  assert manifest.getMainAttributes().getValue('Minimum-Java-Version').equals('1.8')
+  assert manifest.getMainAttributes().getValue('Plugin-Developers').equals('Noam Chomsky:nchomsky:nchomsky@example.com')
+  assert manifest.getMainAttributes().getValue('Plugin-License-Name').equals('MIT License')
+  assert manifest.getMainAttributes().getValue('Plugin-License-Url').equals('https://opensource.org/licenses/MIT')
+  assert manifest.getMainAttributes().getValue('Plugin-ScmUrl').equals('https://github.com/jenkinsci/maven-hpi-plugin')
+  assert manifest.getMainAttributes().getValue('Plugin-Version').startsWith('1.0-SNAPSHOT')
+  assert manifest.getMainAttributes().getValue('Short-Name').equals('verify-it')
+  assert manifest.getMainAttributes().getValue('Specification-Title').equals('MyNewPlugin') // was project.description in previous versions, now project.name
```

Comment:
Core does consume this to display on the installed plugins page, but only if `src/main/resources/index.jelly` is not present. As of #302 we enforce the existence of `src/main/resources/index.jelly`, so changing the semantics of `Specification-Title` will not result in any change in behavior.